ns/install.sh

```diff
@@ -0,0 +1,2 @@
+sudo apt-get install ruby
+sudo gem install erubis
```

Comment:
We should probably create a Gemfile and do a bundle install to get necessary dependencies.